content/id/docs/concepts/scheduling/scheduling-framework.md

```diff
@@ -0,0 +1,251 @@
+di akhir siklus pengikatan.
+
+### Permit
+
+_Plugin_ Permit dipanggil pada akhir siklus penjadwalan untuk setiap Pod
+untuk mencegah atau menunda pengikatan ke Node kandidat. _Plugin_ Permit dapat 
+melakukan salah satu dari ketiga hal ini:
+
+1. ** approve ** \
+     Setelah semua _plugin_ Permit menyetujui sebuah Pod, untuk dikirimkan dalam 
+     pengikatan.
+
+2. ** deny ** \
+     Jika ada _plugin_ Permit yang menolak sebuah Pod, untuk dikembalikan ke 
+     antrian penjadwalan. Hal ini akan memicu _plugin_ [Unreserve](#unreserve).
+
+3. ** wait ** (dengan batas waktu) \
+     Jika _plugin_ Permit menghasilkan "wait", maka Pod disimpan dalam
+     daftar Pod tunggu internal, dan siklus pengikatan Pod ini dimulai tetapi akan langsung di blok
+     sampai mendapatkan [_approved_](#frameworkhandle). Jika waktu tunggu habis, ** wait ** menjadi ** deny **
+     dan Pod dikembalikan ke antrian penjadwalan, yang memicu _plugin_ [Unreserve](#unreserve).
+
+{{< note >}}
+Ketika setiap _plugin_ dapat mengakses daftar "tunggu" Pod-Pod dan menyetujuinya
+(silahkan lihat [`FrameworkHandle`](#frameworkhandle)), kami hanya mengharapkan
+_plugin_ Permit untuk menyetujui pengikatan Pod dalam kondisi "menunggu" yang 
+telah dipesan. Sekali Pod disetujui, akan dikirim ke fase [PreBind](#pre-bind).
+{{< /note >}}
+
+### PreBind {#pre-bind}
+
+_Plugin_ ini digunakan untuk melakukan pekerjaan apa pun yang diperlukan sebelum
+Pod terikat. Sebagai contoh, _plugin_ PreBind dapat menyediakan _network volume_ 
+dan melakukan _mounting_ pada target Node sebelum mengizinkan Pod berjalan di 
+sana.
+
+Jika ada _plugin_ PreBind menghasilkan kesalahan, maka Pod [ditolak](#unreserve)
+dan kembali ke antrian penjadwalan.
+
+### Bind
+
+_Plugin_ ini digunakan untuk mengikat Pod ke Node. _Plugin-plugin_ Bind tidak akan
+dipanggil sampai semua _plugin_ PreBind selesai. Setiap _plugin_ bind dipanggil 
+sesuai urutan saat dikonfigurasi. _Plugin_ Bind dapat memilih untuk menangani 
+atau tidak Pod yang diberikan. Jika _plugin_ Bind memilih untuk menangani Pod, 
+** _plugin_ Bind yang tersisa adalah dilewati **.
+
+### PostBind {#post-bind}
+
+Ini adalah titik ekstensi bersifat informasi. _Plugin-plugin_ PostBind dipanggil
+setelah sebuah Pod berhasil diikat. Ini adalah akhir dari siklus pengikatan, dan
+dapat digunakan untuk membersihkan sumber daya terkait.
+
+### Unreserve
+
+Ini adalah titik ekstensi bersifat informasi. Jika sebuah Pod telah dipesan dan 
+kemudian ditolak di tahap selanjutnya, maka _plugin-plugin_ Unreserve akan 
+diberitahu. _Plugin_ Unreserve harus membersihkan status yang terkait dengan Pod
+yang dipesan.
+
+_Plugin_ yang menggunakan titik ekstensi ini biasanya juga harus digunakan
+[Reserver](#unreserve).
+
+## _Plugin_ API
+
+Ada dua langkah untuk _plugin_ API. Pertama, _plugin_ harus mendaftar dan mendapatkan
+konfigurasi, kemudian mereka menggunakan antarmuka titik ekstensi. Antarmuka 
```

Comment:
```suggestion
konfigurasi, kemudian mereka menggunakan antarmuka (_interface_) titik ekstensi. Antarmuka 
```

The "interface" directly refers to the `Plugin` Go interface. We can put an italicized "interface" to help the readers.